services/validation/duty_validation.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  CannotModifyAssignedDutyError,
+  DocumentNotFoundError,
+} from '../../error_handling/client_errors';
+import { Duty } from '../schemas/duty.zschema';
+
+const validateDutyIsModifiable = (id: string, duty: Duty | null) => {
```

Comment:
id type

Reply:
which type? I didn't set any specific id type

Reply:
create schema for id to be objectId

Reply:
resolved